GHzDACs/Cleanup/dac.py

```diff
@@ -847,50 +861,62 @@ def func():
                     clkinv = not clkinv
                     yield self._setPolarity(chan, clkinv)
                     tries += 1
-    
+
             ans = found, clkinv, PHOF, tries, targetFifo
             returnValue(ans)
+
         return self.testMode(func)
-    
+
     def runBIST(self, cmd, shift, dataIn):
-        """Run a BIST on the given SRAM sequence. (DAC only)"""
+        """
+
```

Comment:
Would be nice to keep the old summary, in addition to the parameter descriptions.